K2Bridge/Visitors/QueryStringClauseVisitor.cs

```diff
@@ -44,10 +44,17 @@ public void Visit(QueryStringClause queryStringClause)
             switch (queryStringClause.ParsedType)
             {
                 case QueryStringClause.Subtype.Term:
-                    var isNumeric = GetIsFieldNumeric(queryStringClause.ParsedFieldName).Result;
-                    if (isNumeric)
+                    var isMatchPhraseFieldType = IsMatchPhraseFieldType(queryStringClause.ParsedFieldName).Result;
+                    if (isMatchPhraseFieldType)
                     {
-                        queryStringClause.KustoQL = $"{queryStringClause.ParsedFieldName} {KustoQLOperators.Equal} {queryStringClause.Phrase}";
+                        var matchPhraseClause = new MatchPhraseClause()
+                        {
+                            FieldName = queryStringClause.ParsedFieldName,
+                            Phrase = queryStringClause.Phrase,
+                        };
+
+                        matchPhraseClause.Accept(this);
```

Comment:
on another PR (not yet open) we are adding support for >= and <=

what happens if the field type is numeric but the pharse is '>=2', this is a current bug.
so you will get to MatchPharseClause and you will need to support >= in addition to Equal.